contract/src/test/kotlin/io/provenance/scope/loan/utility/ContractRequirementsTest.kt

```diff
@@ -0,0 +1,95 @@
+package io.provenance.scope.loan.utility
+
+import io.kotest.assertions.throwables.shouldNotThrow
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.ints.shouldBeGreaterThanOrEqual
+import io.kotest.matchers.ints.shouldBeInRange
+import io.kotest.matchers.ints.shouldBeLessThan
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.types.shouldBeTypeOf
+import io.kotest.property.Arb
+import io.kotest.property.arbitrary.list
+import io.kotest.property.checkAll
+import io.provenance.scope.loan.test.shouldHaveViolationCount
+import io.provenance.scope.loan.test.LoanPackageArbs
+
+class ContractRequirementsTest : WordSpec({
```

Comment:
I have no prior experience with Kotest, but I liked the style of `WordSpec`s the best. Open to discussion on what testing practices and styles we want to follow.